cityofaustin/atd-moped
# Migrate project subprojects table to DataGridPro

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18180

## Testing
**URL to test:** 

[Deploy preview](https://deploy-preview-1474--atd-moped-main.netlify.app/)

**Steps to test:**
1. Go to the summary view of a project.
2. See the subprojects table, test adding a new subproject.
3. Delete that project.
4. Add more projects
5. Test sorting, filtering, adjusting column widths, all the fun datagrid things
6. Project status badge and id should be empty on record add and should populate once you save.
7. Make sure the status badge in the table matches the actual status badge of the project.


---
#### Ship list
